EpsilonLights/Src/main.c

```diff
@@ -289,11 +294,12 @@ static void MX_CAN2_UserInit(void)
 
     CAN_FilterConfTypeDef batteryFilterConfig;
     batteryFilterConfig.FilterNumber = 1; // Use secondary filter bank
+    batteryFilterConfig.FilterScale = CAN_FILTERSCALE_32BIT;
     batteryFilterConfig.FilterMode = CAN_FILTERMODE_IDLIST; // Look for specific can messages
-    batteryFilterConfig.FilterIdHigh = BATTERY_STAT_ERRORS_STDID << 5; // Filter registers need to be shifted left 5 bits
```

Comment:
Why was this removed? We need to be checkin this for the strobe light, what's changed?

Reply:
We added a few things to the comms protocol. Electrical explained that if a fault trips, the Orion contractors will disengage, so the Aux BMS will catch that and notify the lights board.